dockerversion/version_lib.go

```diff
@@ -9,4 +9,5 @@ var (
 	PlatformName          = ""
 	ProductName           = ""
 	DefaultProductLicense = ""
+	BuildKitVersion       = "v0.11.2+unknown"
```

Comment:
@thaJeztah I think it would be handy to also add this to `docker info` in follow-up. WDYT?

Reply:
I feel like we should avoid adding more versions to `docker info` -- the Go psuedo-versions/SHAs presented there are quite unhelpful/confusing to users as they are presented as e.g. 'containerd version: SHA'

I would suggest instead that you add reporting of the BuildKit version to `docker version`, which is designed to be modular and doesn't require a new API to add this information.